content/en/blog/general/apply-vap-using-cli/index.md

```diff
@@ -0,0 +1,222 @@
+---
+date: 2023-09-04
+title: "Applying Validating Admission Policies using Kyverno CLI"
+linkTitle: "Applying Validating Admission Policies using Kyverno CLI"
+author: Mariam Fahmy
+description: "Using Kyverno CLI to apply Validating Admission Policies"
+---
+The [Kyverno Command Line Interface (CLI)](https://kyverno.io/docs/kyverno-cli/) is designed to validate and test policy behavior to resources prior to adding them to a cluster.
+
+One of the Kyverno CLI commands are apply and test commands.
+
+1. The apply command is used to perform a dry run on one or more policies for the given manifest(s).
+2. The test command is used to test a given set of resources against one or more policies to check the desired results.
+
+Kyverno CLI now can be used to apply/test Kubernetes Validating Admission Policies that were first introduced in 1.26.
+
+## Applying Validating Admission Policies using Kyverno apply
```

Comment:
New line between headings and code block fences, please. This keeps the Markdown linters happy.